apps/user/services/authentication.go

```diff
@@ -95,52 +95,102 @@ func (service AuthenticationService) BasicLogin(c *gin.Context) (
 
 // Identify ...
 func (service AuthenticationService) Identify(c *gin.Context) (
+	reSendTime int,
```

Comment:
Can we use a serializer for `reSendTime`?

Reply:
we don't need this as there only one field and it's not gonna change.